### PR TITLE
Events page improvements

### PR DESCRIPTION
**Pull request by BradHN1**
#### Requirements (place an `x` in each `[ ]`)
* [x] I've read and understood the [Contributing Guidelines](/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've tested my changes manually.
* [ ] I've added unit tests to my PR.

##### Transparency ([Project board](https://github.com/orgs/massenergize/projects/7/views/2))
* [x] I've given my PR a meaningful title.
* [x] I linked this PR to the relevant issue.
* [x] I moved the linked issue from the "Sprint backlog" to "In progress" when I started this.
* [x] I moved the linked issue to "Local testing" now that it is ready to merge.

### CODE REVIEW

**Comment by BradHN1** [comment]
See video and picture for new functionality in the ticket #1143

File: src/components/Pages/EventsPage/EventsPageReal.js [hunk 1]
```diff
@@ -126,6 +128,34 @@ class EventsPage extends React.Component {
     const found =
       this.searchIsActiveSoFindContentThatMatch() ||
       applyTagsAndGetContent(this.props.events, this.state.checked_values);
+    console.log(found)
```
Comment (abdullai-t) [inline src/components/Pages/EventsPage/EventsPageReal.js:131]: Please remove log statement

**Review by abdullai-t — APPROVED** [review]
LGTM ✅

**Comment by abdullai-t** [comment]
The test is broken 😞

**Pull request merged**